website/releases/release-0.11.0.md

```diff
@@ -0,0 +1,206 @@
+Similar settings work for Flink. For more details, please refer to `HoodieIndexConfig` in
+the [configurations](/docs/configurations) page.
+
+### CLI Savepoint & Restore
+
+Disaster recovery is a mission critical feature in any production deployment. Especially when it comes to systems that
+store data. Hudi had savepoint and restore functionality right from the beginning for COW tables. In 0.11.0, we have
+added support for MOR tables. More info about this feature can be found in [Disaster Recovery](/docs/next/disaster_recovery).
+
+### Write Commit Callback for Pulsar
+
+Hudi users can use `org.apache.hudi.callback.HoodieWriteCommitCallback` to invoke callback function upon successful
+commits. In 0.11.0, we add`HoodieWriteCommitPulsarCallback` in addition to the existing HTTP callback and Kafka
+callback. Please refer to `org.apache.hudi.utilities.callback.pulsar.HoodieWriteCommitPulsarCallbackConfig` for
+configurations to set.
+
+### HiveSchemaProvider
+
+In 0.11.0, `org.apache.hudi.utilities.schema.HiveSchemaProvider` is added for getting schema from user-defined hive
+tables. This is useful when tailing Hive tables in `HoodieDeltaStreamer` instead of having to provide avro schema files.
+
+## Migration Guide
+
+### Upgrade steps for enabling metadata table
+
+*Before the upgrade, stop all writers on the same table.*
+
+- Deployment Model A: If your current deployment model is single writer and all table services (cleaning, clustering,
+  compaction) are configured to be **inline**, such as Deltastreamer sync-once mode and Spark Datasource with default
+  configs, there is no additional configuration required. Restarting the Single Writer with 0.11.0 is sufficient to
+  safely enable metadata table.
+- Deployment Model B: If your current deployment model is single writer along with async table services (such as
+  cleaning, clustering, compaction) running in the same process, such as `HoodieDeltaStreamer` continuous mode writing
+  MOR table, Spark streaming (where compaction is async by default), and your own job setup enabling async table
+  services inside the same writer, it is a must to have the optimistic concurrency control, the lock provider, lazy
+  failed write clean policy configured before enabling metadata table. More details can be found [here](/docs/metadata).
+  Failing to follow the configuration guide leads to loss of data.
+- Deployment Model C: If your current deployment model is [multi-writer](/docs/concurrency_control) along
+  with [a lock provider](/docs/concurrency_control/#enabling-multi-writing) and other required configs set for every
+  writer, there is no additional configuration required. You can bring up the writers with 0.11.0 sequentially. Applying
+  the proper configurations to only partial writers leads to loss of data from the inconsistent writer. So, ensure you
+  enable metadata table across all writers.
+
```

Comment:
can we add one more line to clarify that, lock provider is not mandatory for anyone running async table services. Only when are enabling metadata data (which is enabled by default), lock providers are mandatory. If someone explicitly disables metadata table, they don't need to configure lock providers.

Reply:
this is clarified at the top paragraph. https://github.com/apache/hudi/pull/5466#discussion_r861945249